web/src/beta/features/Editor/Map/LayerStylePanel/PresetLayerStyle/index.tsx

```diff
@@ -113,14 +113,14 @@ const PresetLayerStyle: FC<PresetLayerStyleProps> = ({
         },
         {
           id: "threedTiles",
-          title: t("3D Tiles"),
+          title: "3D Tiles", // TODO: We will add translation t("3D Tiles") later
```

Comment:
```suggestion
          title: "3D Tiles", 
```
Current decision is using english here, no need to add this TODO i think.